packages/fuel-indexer-tests/tests/e2e/graphql_api_postgres.rs

```diff
@@ -71,8 +73,9 @@ async fn test_can_return_query_response_with_all_fields_required_postgres() {
 async fn test_can_return_query_response_with_nullable_fields_postgres() {
     let fuel_node_handle = tokio::spawn(setup_example_test_fuel_node());
 
-    let mut srvc = indexer_service_postgres().await;
-    let api_app = api_server_app_postgres().await;
+    let test_db = TestPostgresDb::new().await;
+    let mut srvc = indexer_service_postgres(Some(&test_db.url)).await;
```

Comment:
This logic of `indexer_service_postgres(Some(&test_db.url))` and `api_server_app_postgres(Some(&test_db.url))` with the `Some()` arg seems pretty repetitive, is there some way we can hide that logic given that _all_ of these take the exact same arg of `Some(&test_db.url)`